codesy/templates/401.html

```diff
@@ -1 +1,13 @@
-<a href="https://{{ current_site.domain }}">Sign in to codesy.io</a>.
+{% load staticfiles %}
+<link rel="stylesheet" href="//{{current_site.domain}}{% static "css/styles.css" %}">
+<link rel="stylesheet" href="//cdn.jsdelivr.net/pure/0.5.0/pure-min.css">
```

Comment:
Can you move these lines into a separate template file and use `{% include %}` to include it in both `401.html` and `bid.html`? If not, just file an issue for that and I'll r+ and merge this.